src/native/libs/System.Globalization.Native/CMakeLists.txt

```diff
@@ -218,3 +218,14 @@ if(CLR_CMAKE_TARGET_WIN32)
         install_static_library(System.Globalization.Native.Aot aotsdk nativeaot)
     endif()
 endif()
+
+if((CLR_CMAKE_TARGET_BROWSER OR CLR_CMAKE_TARGET_WASI) AND DEFINED CMAKE_ICU_DIR)
+    file(TO_CMAKE_PATH "${CMAKE_ICU_DIR}" CMAKE_ICU_DIR)
+    file(GLOB ICU_DATA_AND_LIB_FILES "${CMAKE_ICU_DIR}/lib/*.dat")
+    list(APPEND ICU_DATA_AND_LIB_FILES
+        "${CMAKE_ICU_DIR}/lib/libicuuc.a"
+        "${CMAKE_ICU_DIR}/lib/libicui18n.a"
+        "${CMAKE_ICU_DIR}/lib/libicudata.a"
```

Comment:
This includes icu data regardless of `InvariantGlabalization` we would have to fix that. I'm not clear right now if this also impacts Mono download size

Reply:
I'm worried about it being included in `System.Globalization.Native-Static` that we link for Mono/browser too

Reply:
Well, this change doesn't affect the linking of that library (or its dependencies).

Reply:
I just learned that `libicudata.a` is just a empty stub. All good.